sdk/batch/azure-batch/azure/batch/custom/custom_errors.py

```diff
@@ -8,12 +8,30 @@ class CreateTasksErrorException(Exception):
     """ Aggregate Exception containing details for any failures from a task add operation.
 
     :param str message: Error message describing exit reason
-    :param [~TaskAddParameter] pending_task_list: List of tasks remaining to be submitted.
+    :param [~TaskAddParameter] pending_tasks: List of tasks remaining to be submitted.
     :param [~TaskAddResult] failure_tasks: List of tasks which failed to add
     :param [~Exception] errors: List of unknown errors forcing early termination
     """
-    def __init__(self, message, pending_task_list=None, failure_tasks=None, errors=None):
-        self.message = message
-        self.pending_tasks = list(pending_task_list)
+    def __init__(self, pending_tasks=None, failure_tasks=None, errors=None):
+        if failure_tasks and errors:
+            self.message =  "Multiple errors encountered. " \
+                            "Check the `failure_tasks` and `errors` properties for additional details"
+        if errors:
+            if len(errors) > 1:
+                self.message = "The SDK encountered an internal error. " \
```

Comment:
Is this really an internal SDK error or a server error?

Reply:
Its either a) A single task is too large or b) a non-server error on the add-collection(not the inner tasks